templates/engage/microsoft-active-directory.html

```diff
@@ -1,8 +1,4 @@
-{% extends "engage/base_engage.html" %}
-
-{% block title %}How to integrate Ubuntu Desktop with Active Directory{% endblock %}
-
-{% block meta_description %}Ubiquitous use of Microsoft tools coupled with increasing popularity of open source Linux software for enterprise presents new challenges for non-Microsoft operating systems that require seamless integration with Active Directory for authentication and identity management.{% endblock %}
+{% extends_with_args "engage/base_engage.html" with title="How to integrate Ubuntu Desktop with Active Directory" meta_image="https://assets.ubuntu.com/v1/563fa732-ubuntu+and+active+directory.svg" meta_description="Ubiquitous use of Microsoft tools coupled with increasing popularity of open source Linux software for enterprise presents new challenges for non-Microsoft operating systems that require seamless integration with Active Directory for authentication and identity management." %}
```

Comment:
This image is white and so wont show up well socially. Maybe remove it for now or take a screenshot of the size and upload a png?